my_project/tab_psy_chart/app_psy_chart.py

```diff
@@ -308,26 +313,25 @@ def update_psych_chart(
     elif var == "Frequency":
         var_color = ["rgba(255,255,255,0)", "rgb(0,150,255)", "rgb(0,0,150)"]
     else:
-        var_unit = mapping_dictionary[var]["unit"]
+        var_unit = mapping_dictionary[var][si_ip]["unit"]
 
         var_name = mapping_dictionary[var]["name"]
 
         var_color = mapping_dictionary[var]["color"]
 
     if global_local == "global":
         # Set Global values for Max and minimum
-        var_range_x = mapping_dictionary["DBT"]["range"]
-        hr_range = [0, 0.03]
-        var_range_y = hr_range
+        var_range_x = mapping_dictionary["DBT"][si_ip]["range"]
+        var_range_y = mapping_dictionary["hr"][si_ip]["range"]
 
```

Comment:
I changed the range according to the mapping_dictionary, so it can change when the unit system changes.